index.src.html

```diff
@@ -972,22 +976,14 @@ <h4 id="fetch-api">Fetch API</h4>
           IPAddressSpace targetAddressSpace;
         };
       </pre>
-  - The <a constructor for=Request lt="Request(input, init)"><code>new
-    Request(<var ignore=''>input</var>, |init|)</code></a> is
-    appended with the following step right before setting [=this=]'s [=request=]
-    to |request|:
-    1.  If |init|["{{RequestInit/targetAddressSpace}}"] [=map/exists=], and
-        |request|'s [=request/client=] is a [=secure context=], then switch on
-        |init|["{{RequestInit/targetAddressSpace}}"]:
```

Comment:
@johnathan79717 I don't think we checked or we should check secure context in Request. Just forward it and check in mixed content check is fine.

Reply:
As discussed offline, we keep the definitions of request and requestInit which only set request's target IP address space when requestInit's is private or local.